std/range/package.d

```diff
@@ -4663,6 +4667,10 @@ private string lockstepMixin(Ranges...)(bool withIndex, bool reverse)
 
    By default $(D StoppingPolicy) is set to $(D StoppingPolicy.shortest).
 
+   Limitations: Compiler cannot currently infer purity, safety, use of garbage
```

Comment:
Ditto.

Reply:
How so? Unlike the previous one, this one is already in the negative can-not form instead of has-to.

Reply:
I mean, use the attributes directly. Purity -> `pure`, safety -> `@safe`, etc.